src/app/app.component.html

```diff
@@ -19,6 +19,24 @@
       <span>FAQ</span>
     </button>
   </mat-menu>
+<!--
+  <button mat-button [matMenuTriggerFor]="service">Our Services</button>
+  <mat-menu #service="matMenu">
+    <button mat-menu-item routerLink="/our-services">
+      <mat-icon>mood</mat-icon>
+      <span>General Dentistry</span>
+    </button>
+    <button mat-menu-item routerLink="/our-services">
+      <mat-icon>mood</mat-icon>
+      <span>Cosmetic Dentistry</span>
+    </button>
+    <button mat-menu-item routerLink="/our-services">
+      <mat-icon>mood</mat-icon>
+      <span>Restorative Dentistry</span>
+    </button>
+  </mat-menu>
+-->
```

Comment:
Try to minimize commiting commented code

Reply:
I have this part here so we can discuss during the meeting if we should have the tab under "About Us" or a new separate tab

Reply:
Ah ok sorry 😅